lib/cosmos/interfaces/protocols/template_protocol.rb

```diff
@@ -41,7 +48,8 @@ def initialize(
       sync_pattern = nil,
       fill_fields = false,
       response_timeout = 5.0,
-      response_polling_period = 0.02
+      response_polling_period = 0.02,
+      error_action = 'LOG'
```

Comment:
Should this just be "disconnect_on_error" with a true false? I log it every time no matter what. I can't remember what we decided.

Reply:
I think disconnect_on_error would be better.

Reply:
Or raise_exceptions true/false?

Reply:
@jmthomas Incorporate this change and this ticket looks good.